PyRoute/Star.py

```diff
@@ -231,6 +231,7 @@ def _parse_line_into_star_core(star, line, sector, pop_code, ru_calc):
             star.popCode = 12
 
         star.tradeCode = TradeCodes(data[3].strip())
+        star.tradeCode.trim_ill_formed_residual_codes()
```

Comment:
Why is this done as a separate step? I would think the trim should be part of the init for the class.

Reply:
Rolled into ctor